deepchem/feat/graph_features.py

```diff
@@ -263,14 +290,30 @@ class WeaveFeaturizer(Featurizer):
 
   name = ['weave_mol']
 
-  def __init__(self):
+  def __init__(self, graph_distance=True, explicit_H=None):
```

Comment:
It's like if you call this function with explicit_H=False, then we will not check hydrogens. But if you do not specify, we will check by default.